product_docs/docs/pgd/5/consistency/crdt/01_crdt_usage.mdx

```diff
@@ -138,10 +140,9 @@ SELECT id, counter FROM crdt_delta_example WHERE id = 1;
 (1 row)
 ```
 
-With a regular `integer` column, the result is `2`. But when you update the row with a delta CRDT counter, you start with the OLD row version, make a NEW row version, and send both to the remote node. There, compare them with the version found there (e.g., the LOCAL version). Standard CRDTs merge the NEW and the LOCAL version, while delta CRDTs compare the OLD and NEW versions and apply the delta
+With a regular `integer` column, the result is `2`. But when you update the row with a delta CRDT counter, you start with the OLD row version, make a NEW row version, and send both to the remote node. There, compare them with the version found there (for example, the LOCAL version). Standard CRDTs merge the NEW and the LOCAL version. Delta CRDTs compare the OLD and NEW versions and apply the delta
```

Comment:
Are the u/c words here being used for emphasis or is this referring to something appears in code? If emphasis, we don't use all caps for this purpose or really any kind of emphasis, and I would lower case them. If referring to something that appears in the code, that's fine.

Reply:
Okay, will leave.